src/common/components/select-repository-row/index.js

```diff
@@ -16,32 +16,32 @@ class SelectRepositoryRow extends Component {
     // TODO tidy up when we get rid of prefixes
     const isChecked = repository.isSelected || isEnabled;
     const isFetching = repository.isFetching;
-    const isDisabled = isEnabled || isFetching;
+    const isDisabled = isEnabled || isFetching || !repository.isAdmin;
```

Comment:
This is pre-existing, I know, but it might be nice to sort out slightly different names for either or both of `isEnabled` (is repository already enabled as a snap build) and `isDisabled` (should this repository row be unavailable for selection) so that it's less terribly confusing.